model/notify_admin.go

```diff
@@ -22,6 +22,7 @@ const (
 	PaidFeatureUnlimitedBoardcards     = MattermostPaidFeature("mattermost.feature.unlimited_board_cards")
 	PaidFeatureAllProfessionalfeatures = MattermostPaidFeature("mattermost.feature.all_professional")
 	PaidFeatureAllEnterprisefeatures   = MattermostPaidFeature("mattermost.feature.all_enterprise")
+	UpgradeDowngradeWorkspace          = MattermostPaidFeature("mattermost.feature.upgrade_downgrade_workspace")
```

Comment:
Is this for a related change?

Reply:
All the changes are for [the same task](https://mattermost.atlassian.net/browse/MM-45580), without this and the delinquency_since I can't finish it